src/middlewared/middlewared/plugins/account.py

```diff
@@ -334,6 +334,7 @@ async def do_update(self, pk, data):
             user['group'] = group['id']
 
         await self.__common_validation(verrors, data, 'user_update', pk=pk)
+        must_change_pdb_entry = True if data.get("password") or data.get("username") or data.get("locked") else False
```

Comment:
Should comparison with original state be used instead? IIRC, `username` will also be present in UI requests. Also, `locked` transition from `True` to `False` won't be captured like this.

Reply:
Indeed. I apparently had a brain-fart while writing that.